registrations/tasks.py

```diff
@@ -1025,28 +1025,7 @@ def run(self, registration_id, **kwargs):
             return
 
         json_doc = self.build_jembi_json(registration)
-        try:
-            result = requests.post(
-                self.URL,
-                headers={"Content-Type": "application/json"},
-                data=json.dumps(json_doc),
-                auth=(settings.JEMBI_USERNAME, settings.JEMBI_PASSWORD),
-                verify=False,
-            )
-            result.raise_for_status()
-            return result.text
-        except (HTTPError,) as e:
-            # retry message sending if in 500 range (3 default retries)
-            if 500 < e.response.status_code < 599:
-                raise self.retry(exc=e)
-            else:
-                self.log.error("Error when posting to Jembi. Payload: %r" % (json_doc))
-                raise e
-        except (Exception,) as e:
-            self.log.error(
-                "Problem posting Registration %s JSON to Jembi" % (registration_id),
-                exc_info=True,
-            )
+        request_to_jembi_api(self.URL, json_doc)
```

Comment:
I would change this to: `request_to_jembi_api.delay(self.URL, json_doc)`, the delay part makes sure it is not executed directly but put on the queue.

Reply:
because in the run function that executes this, the task is done by task.delay(), wont this result in a double delay?

Reply:
No, it just means the main task can put this one on the queue and complete. Then the celery worker will pick up the `request_to_jembi_api` task and execute.